docker/fetcher/README.md

```
@@ -0,0 +1,97 @@
+docker run \
+ -p 9000:9000 \
+ ziplineai/chronon-fetcher:latest
+```
+
+### Additional Configuration & Environment Variables
+
+There's some additional environment variables that the Docker container can be started up with (using the `-e "VAR=VALUE"` syntax)
+
+|Environment Variable |Description                    |Default Value          |
+|---------------------|-------------------------------|-----------------------------|
+|**Google Cloud Configuration**
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)